### PR TITLE
Bug 1183970 - Add a test for removing cells from top sites

### PR DESCRIPTION
**Pull request by wesj**

### CODE REVIEW

**Comment by fxios-buildasaur** [comment]
Result of integration 1
Integration took 18 seconds.
? build errors: checkout-error

File: UITests/TopSitesTests.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```
Comment (thebnich) [inline UITests/TopSitesTests.swift]: Nit: align

File: UITests/TopSitesTests.swift [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+class TopSitesTests: KIFTestCase {
+    private var webRoot: String!
+
+    override func setUp() {
+        webRoot = SimplePageServer.start()
+    }
+
+    /**
+    * Tests for removing a site from top sites
+    */
```
Comment (thebnich) [inline UITests/TopSitesTests.swift]: nit

File: UITests/TopSitesTests.swift [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        tester().tapViewWithAccessibilityLabel("Top sites")
+        tester().tapViewWithAccessibilityLabel("Page 1")
+
+        // Verify that Page 1 loads
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+    }
+
+    /**
+    * Tests for removing a site from top sites
+    */
```
Comment (thebnich) [inline UITests/TopSitesTests.swift]: nit

**Comment by thebnich** [comment]
This looks OK on its own, but this could fail when run as the test suite. Right now, we don't clear history between each test run, so the top sites panel will be full of thumbnails from past tests when this one begins :(

Maybe this would be a good time to figure out how to do that...

**Comment by wesj** [comment]
lttm

File: Client/Frontend/Widgets/ThumbnailCell.swift [hunk 4]
```diff
@@ -127,6 +127,7 @@ class ThumbnailCell: UICollectionViewCell {
 
     lazy var removeButton: UIButton = {
         let removeButton = UIButton()
+        removeButton.accessibilityLabel = NSLocalizedString("Remove page", tableName: "topSites", comment: "Button shown in editing mode to remove this site from the top sites panel.")
```
Comment (st3fan) [inline Client/Frontend/Widgets/ThumbnailCell.swift]: Please remove `tableName`. This can go into the global strings table.

**Comment by sleroux** [comment]
I believe we now have a way of clearing history after test runs so we can probably update this to get it working.

**Comment by sleroux** [comment]
Actually I merged this already in another PR: https://github.com/mozilla/firefox-ios/pull/1044. Closing this PR.

**Pull request closed without merging**